### PR TITLE
Update the image for API trigger and the source view

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: en/docs/services/create-your-first-service.md [hunk 1]
```diff
@@ -134,14 +134,13 @@ import ballerinax/covid19;
...
-    resource function get stats/[string country](http:Caller caller, http:Request request) returns error? {
+    resource function get stats/[string country](http:Caller caller) returns error? {
 
         covid19:Client covid19Client = check new ();
-
         covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
         var totalCases = statusByCountry?.cases ?: 0d;
-        worldbank:Client worldBankClient = check new ();
-        worldbank:CountryPopulation[] populationByCountry = check worldBankClient->getPopulationByCountry(country);
+        worldbank:Client worldbankEndpoint = check new ();
```
Comment (anupama-pathirage) [inline en/docs/services/create-your-first-service.md:142]: Why u change this? Text says to use `worldBankClient` as the connection name.

Comment (RukshiW) [reply en/docs/services/create-your-first-service.md]: Corrected.

**Review by Nashaath — APPROVED** [review]

File: en/docs/services/create-your-first-service.md [hunk 2]
```diff
@@ -134,14 +134,14 @@ import ballerinax/covid19;
...
+    resource function get stats/[string country](http:Caller caller) returns error? {
 
         covid19:Client covid19Client = check new ();
-
         covid19:CovidCountry statusByCountry = check covid19Client->getStatusByCountry(country);
         var totalCases = statusByCountry?.cases ?: 0d;
-        worldbank:Client worldBankClient = check new ();
-        worldbank:CountryPopulation[] populationByCountry = check worldBankClient->getPopulationByCountry(country);
+        worldbank:Client worldbankClient = check new ();
+
```
Comment (anupama-pathirage) [inline en/docs/services/create-your-first-service.md:143]: Why addign a new line here?

Comment (RukshiW) [reply en/docs/services/create-your-first-service.md]: Removed the line.

**Pull request merged**